src/Reports/AnnualMaximumAndMinimum/AnnualMaximumAndMinimum.csproj

```diff
@@ -94,9 +94,8 @@
       <HintPath>$(SolutionDir)\packages\System.Numerics.Vectors.4.5.0\lib\net46\System.Numerics.Vectors.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="System.Runtime.CompilerServices.Unsafe, Version=4.0.4.1, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <HintPath>$(SolutionDir)\packages\System.Runtime.CompilerServices.Unsafe.4.5.3\lib\net461\System.Runtime.CompilerServices.Unsafe.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="System.Runtime.CompilerServices.Unsafe, Version=4.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\System.Runtime.CompilerServices.Unsafe.4.6.0\lib\netstandard2.0\System.Runtime.CompilerServices.Unsafe.dll</HintPath>
```

Comment:
@josef-mixan-ai I think you missed a `$(SolutionsDir)` here and a few other places.

Essentially, this git grep sequence needs to return no results

```sh
git grep "<HintPath>" -- "*.csproj" | grep packages | grep -v "(SolutionDir)"
```
![image](https://user-images.githubusercontent.com/54773/188988076-0c6fab7d-fdd6-4fb9-81ba-19a991e654fb.png)

Reply:
@DougSchmidt-AI  I did miss al lot of them actually. It all should be updated now.